secret/secret.go

```diff
@@ -0,0 +1,96 @@
+package secret
+
+import (
+	"log"
+	"os"
+
+	"github.com/SUSE/scf-secret-generator/model"
+	"github.com/SUSE/scf-secret-generator/password"
+	"github.com/SUSE/scf-secret-generator/ssh"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+)
+
+// The name of the secret stored in the kube API
+const SECRET_NAME = "secret"
+
+func GetSecrets() corev1.SecretInterface {
```

Comment:
Maybe call this `GetSecretInterface` instead. I found it confusing that you call `GetSecrets` followed by `GetOrCreateSecret`

Reply:
Fixed in a6a5a76